docs/rest_api.rst

```diff
@@ -528,14 +528,23 @@ Transfers
    :statuscode 409: If the address or the amount is invalid or if there is no path to the target
    :statuscode 500: Internal Raiden node error
 
+
```

Comment:
Why are there 11 blank lines here? Also there are some trailing spaces and tabs on a couple of otherwise unchanged lines that should be removed before merging.

Reply:
I'll search and remove them.